### DISCUSSION TITLE
Neautral creatures' growth after battle

### DISCUSSION THREAD

**Discussion by Branikolog** [ihhub/fheroes2#6339] [Ideas]
When a hero attacks neutrals and runs away, the remained creatures continue growing. But growth of (ex: half-killed) creatures makes them uncompetitive till the end of the game.
My proposal is to rise the growth of that creatures till the moment, they reached the amount of creatures, they suppose to have... Let's say 2 or 3 weeks after the battle creatures should grow to the same amount, as if they didn't fight.

**Edit:**
Or at least leave creature growth as it was before the fight.

**Comment by ihhub** [comment]
I'm making it as a discussion as it is just a proposal to modify the existing logic.

**Comment by LeHerosInconnu** [comment]
Hello everyone,

I think that growth should not be changed from the standard growth.

Wandering creatures give experience points and can also be recruited with the Diplomacy secondary skill.
So a higher growth gives more experience points and allows to recruit more wandering creatures.
Also, the fact that they regenerate faster than normal can cause a problem with being locked up (human player or even the AI).
For example, the hero decimates part of the group of wandering creatures, but can't defeat them. Another hero can defeat them later.
AI heroes often do this in the original game.
Now, if the growth is changed from the standard growth, it is more difficult and the player (human or AI) can get stuck in the starting area.
